.github/workflows/test.yml

```
@@ -35,6 +35,10 @@ jobs:
       run: |
         python -m pip install --upgrade pip wheel
         pip install . -r requirements.txt
+        pip install -e .
```

This installs the actual `eegnb` package
```
 Installing collected packages: eeg-notebooks
  Attempting uninstall: eeg-notebooks
    Found existing installation: eeg-notebooks 0.2
    Uninstalling eeg-notebooks-0.2:
      Successfully uninstalled eeg-notebooks-0.2
  Running setup.py develop for eeg-notebooks
Successfully installed eeg-notebooks
```

Without this, you would receive an error like: 
```

Run eegnb --help
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.6.12/x64/bin/eegnb", line 5, in <module>
    from eegnb.cli.__main__ import main
ModuleNotFoundError: No module named 'eegnb.cli'
Error: Process completed with exit code 1.
```
when running `eegnb --help` (see this failed Github Actions run: https://github.com/NeuroTechX/eeg-notebooks/runs/1500971022?check_suite_focus=true )

---

The latter error is actually an issue with how the `cli` module isn't detected by `find_packages()` in `setup.py` (it needs a `__init__.py`).

The `-e` flag works around this by doing a symlink install, but shouldn't be needed for normal instas.

---

yeah, there is additional work to be done to get the installation in a cleaner, better place. Albeit the first objective for me is to get Linux builds 'working' first, then iterate on that.

---

@jnaulty If you just run `touch eegnb/cli/__init__.py` and commit that it'd fix the issue. Might as well do it now.

Then you could also:

```suggestion
        pip install .
```